### DISCUSSION TITLE
Possible to merge 2 separate mtx files ?

### DISCUSSION THREAD

**Discussion by meghamsh738** [satijalab/seurat#4667] [General]
Hello! I have been practicing to do normal analysis and was wondering if I could merge the results of 2 separate .mtx files ([4kpbmc ](https://support.10xgenomics.com/single-cell-gene-expression/datasets/2.1.0/pbmc4k)and [8kpbmc](https://support.10xgenomics.com/single-cell-gene-expression/datasets/2.1.0/pbmc8k)). My final goal is to try to combine data from male and female cells in separate wells and try to find out if there are any differences between them. I'm also having trouble understanding how to load my two .mtx files into the code

**Comment by royfrancis** [comment]
How about like this?

```
paths <- c("./data/raw/sample1/","./data/raw/sample2")
names(paths) <- c("control","treated")
s <- CreateSeuratObject(counts=Read10X(data.dir=paths))
```
